zgw_consumers/forms.py

```diff
@@ -0,0 +1,28 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+from OpenSSL import crypto
+
+from .models import Certificate
+
+
+class CertificateAdminForm(forms.ModelForm):
+    class Meta:
+        model = Certificate
+        fields = ("label", "type", "public_certificate", "private_key")
```

Comment:
when using the admin form it's custom to use `fields = "__all__"` and control the fields themselves in the admin class (readonly, `fieldsets`...)